telemetry/definitions/commonDefinitions.json

```diff
@@ -163,6 +180,31 @@
         }
     ],
     "metrics": [
+        {
+            "name": "apigateway_copyUrl",
+            "description": "Copying an API Gateway remote URL",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "apigateway_invokeLocal",
+            "description": "Invoking one simulated API Gateway call using the SAM cli",
+            "metadata": [
+                { "type": "runtime", "required": false },
+                { "type": "httpMethod", "required": false },
+                { "type": "result" },
+                { "type": "debug" }
+            ]
+        },
+        {
+            "name": "apigateway_invokeRemote",
+            "description": "Calling a remote API Gateway",
+            "metadata": [{ "type": "result" }, { "type": "httpMethod", "required": false }]
+        },
+        {
+            "name": "apigateway_startLocalServer",
```

Comment:
Is this a passive event?

Reply:
Nope, it requires user input (the action does not exist today yet)